src/components/Visualization.tsx

```diff
@@ -262,8 +262,8 @@ const Visualization = ({ toggleCatalog }: IVisualization) => {
           data-testid={'react-flow-wrapper'}
           ref={reactFlowWrapper}
           style={{
-            width: window.innerWidth,
-            height: window.innerHeight - 77,
+            width: '100vw',
+            height: 'calc(100vh - 77px )',
```

Comment:
for another time: might be nice to provide a variable/constant for the 77px to know from where it si coming from

Reply:
agree with @apupier on this, there are different parts of the app that could benefit from this, so I'll look into it